commands/attr.go

```diff
@@ -0,0 +1,177 @@
+package commands
+
+import (
+	"fmt"
+	"path"
+	"strconv"
+
+	"github.com/ipfs/iptb/testbed"
+	"github.com/ipfs/iptb/testbed/interfaces"
+
+	cli "github.com/urfave/cli"
+)
+
+var AttrCmd = cli.Command{
+	Category: "ATTRIBUTES",
+	Name:     "attr",
+	Usage:    "get, set, list attributes",
+	Subcommands: []cli.Command{
+		AttrSetCmd,
+		AttrGetCmd,
+		AttrListCmd,
+	},
+}
+
+var AttrSetCmd = cli.Command{
+	Name:      "set",
+	Usage:     "set an attribute for a node",
+	ArgsUsage: "<node> <attr> <value>",
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "save",
+			Usage: "saves attribute value to nodespec",
+		},
+	},
+	Action: func(c *cli.Context) error {
+		flagRoot := c.GlobalString("IPTB_ROOT")
+		flagTestbed := c.GlobalString("bench")
+		flagSave := c.Bool("save")
+
+		if c.NArg() != 3 {
+			return NewUsageError("set takes exactly 3 argument")
+		}
+
+		argNode := c.Args()[0]
+		argAttr := c.Args()[1]
+		argValue := c.Args()[2]
+
+		i, err := strconv.Atoi(argNode)
+		if err != nil {
+			return fmt.Errorf("parse err: %s", err)
+		}
+
+		tb := testbed.NewTestbed(path.Join(flagRoot, "benches", flagTestbed))
+
+		if flagSave {
```

Comment:
Should we first attempt to set the attribute on the node before persisting it to the nodespec file?